backstory/src/test/java/com/google/sps/images/AnnotatedImageTest.java

```diff
@@ -167,12 +191,59 @@ public void getLabelDescriptionsNonEmptyLabelAnnotations() throws IllegalArgumen
       throw exception;
     }
 
-    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations);
+    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations, emptyLandmarkAnnotations);
     List<String> actualLabelDescriptions = annotatedImageActual.getLabelDescriptions();
     List<String> expectedLabelDescriptions = new ArrayList<>();
-    expectedLabelDescriptions.add("DescriptionOne");
-    expectedLabelDescriptions.add("DescriptionTwo");
+    expectedLabelDescriptions.add(firstAnnotationDescription);
+    expectedLabelDescriptions.add(secondAnnotationDescription);
 
     assertEquals(expectedLabelDescriptions, actualLabelDescriptions);
   }
+
+  /**
+   * public List<String> getLandmarkDescriptions();
+   *  landmarkAnnotations is empty.
+  */
+  @Test
+  public void getLandmarkDescriptionsEmptyLandmarkAnnotations() throws IllegalArgumentException, IOException {
+    byte[] rawImageData;
+    try {
+      rawImageData = getBytesFromImageReference(
+        "src/test/java/com/google/sps/images/data/dogRunningOnBeach.jpg", "jpg");
+    } catch (IOException exception) {
```

Comment:
Seems like try/catch is not needed, since this method declares that it throws IOException.

Reply:
I agree. I'll fix this everywhere.

Reply:
Done